package.json

```diff
@@ -319,14 +319,21 @@
     ]
   },
   "scripts": {
+    "postinstall": "husky install",
     "vscode:prepublish": "npm run compile",
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./"
   },
   "devDependencies": {
     "@types/node": "^16.11.7",
     "@types/vscode": "^1.1.59",
+    "husky": "^8.0.3",
+    "lint-staged": "^13.2.1",
+    "prettier": "^2.8.7",
     "typescript": "^4.4.4"
   },
-  "dependencies": {}
+  "dependencies": {},
+  "prettier": {
+    "printWidth": 100
```

Comment:
Let's just finish the rest of the config.

Determine if we are using tabs or spaces, and choose it.

semi: true
singleQuote: false
quoteProps: as-needed
jsxSingleQuote: false
trailingCommas: all
bracketSpacing: true
bracketSameLine: false
arrowParens: true
wrapProse: always
endOfLine: lf

Reply:
spaces